internal/sysfs/poll.go

```diff
@@ -0,0 +1,35 @@
+package sysfs
+
+import (
+	"time"
+
+	"github.com/tetratelabs/wazero/experimental/sys"
+)
+
+// Poll exposes the poll(2) syscall.
```

Comment:
please follow the documentation style of File.PollRead including how it uses links and refers to functions like `poll` not poll(2). I forget who said this, but there's an implicit goal to write as if everything is written by the same person. This means same style, tests, documentation form. This makes it easier to navigate and also easier to tell if other code was looked at etc (e.g. if not you could ask.. did you look at X which is similar?) It doesn't prevent changing style. For example, I went back and changed all links to better open group ones earlier, then all, not just a few we coherently done. So, general rule of thumb is to write like the others, then if something becomes lifted into the main API, no one needs to re-do it, also.